vsintegration/src/FSharp.Editor/Navigation/FindUsagesService.fs

```diff
@@ -123,7 +123,9 @@ type internal FSharpFindUsagesService
                             |> Async.Parallel
                             |> liftAsync
 
-                        return symbolUses |> Array.concat
+                        // FCS may return several `FSharpSymbolUse`s for same range, which have different `ItemOccurrence`s (Use, UseInAttribute, UseInType, etc.)
```

Comment:
Any idea why that happens with Units of measure?  Without knowing the internals, it sounds like a bit of a deeper issue.  Either way, happy that fixing #2387 is a simple change.